airbyte-db/lib/src/main/resources/configs_database/schema2.sql

```diff
@@ -0,0 +1,218 @@
+CREATE
+    TYPE "source_type" AS enum(
+        'api',
+        'file',
+        'database',
+        'custom'
+    );
+
+CREATE
+    TYPE "actor_type" AS enum(
+        'source',
```

Comment:
@cgardens should we start using the official v2 names?

Reply:
i think we should handle it when we work on that protocol v2 project. until then these enum names are more clear and correct when describing what's actually there.